.github/workflows/ci.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - name: Find pip cache dir
         id: pip-cache
-        run: echo "::set-output name=dir::$(pip cache dir)"
+        run: echo "dir=$(pip cache dir)" >> $GITHUB_OUTPUT
```

Comment:
Nit: this step and the next one could be removed: setup-python now supports pip cache using with-arguments:
```
          cache: 'pip'
          cache-dependency-path: 'requirements*.txt'
```

Reply:
Oh, nice. Thanks for pointing this out! I just pushed an extra patch to make this change.